tensorflow_serving/model_servers/BUILD

```diff
@@ -141,6 +141,15 @@ py_test(
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.data-00000-of-00001",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.index",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.meta",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.data-00000-of-00001",
```

Comment:
That was an oversight on my part. I had originally planned on doing both SessionBundle and a SavedModel tests - but then I realised later that the saved model version was in the tensorflow code base...=P good eye!